### ISSUE TITLE
Can't add a new record

### ISSUE THREAD

**Issue by levkovich2806** [depyronick/nestjs-clickhouse#13]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

I try to add a new record via POSTMAN

### Minimum reproduction code

https://github.com/levkovich2806/testClickHouse

### Steps to reproduce

1. create table 
```

CREATE TABLE messages
                         (
                             user_id UInt32,
                             message String
                         )
                         ENGINE = MergeTree()
                         PRIMARY KEY (user_id)
```

2. yarn start
3. send POST request with JSON data
`
curl --header "Content-Type: application/json" --request POST --data '{"user_id":1,"message":"xyz"}' http://localhost:3000`

### Expected behavior

I want to add new record to a table. But I have an error

`Code: 62. DB::Exception: Syntax error: failed at position 1 ('{'): {"user_id":1,"message":"xyz"}. Expected one of: Query, Query with output, EXPLAIN, SELECT query, possibly with UNION, list of union elements, SELECT query, subquery, possibly with UNION, SELECT subquery, SELECT query, WITH, FROM, SELECT, SHOW CREATE QUOTA query, SHOW CREATE, SHOW [FULL] [TEMPORARY] TABLES|DATABASES|CLUSTERS|CLUSTER 'name' [[NOT] [I]LIKE 'str'] [LIMIT expr], SHOW, SHOW ENGINES query, SHOW ENGINES, EXISTS or SHOW CREATE query, EXISTS, DESCRIBE FILESYSTEM CACHE query, DESCRIBE, DESC, DESCRIBE query, SHOW PROCESSLIST query, SHOW PROCESSLIST, CREATE TABLE or ATTACH TABLE query, CREATE, ATTACH, REPLACE, CREATE DATABASE query, CREATE VIEW query, CREATE DICTIONARY, CREATE LIVE VIEW query, CREATE WINDOW VIEW query, ALTER query, ALTER TABLE, ALTER LIVE VIEW, ALTER DATABASE, RENAME query, RENAME TABLE, EXCHANGE TABLES, RENAME DICTIONARY, EXCHANGE DICTIONARIES, RENAME DATABASE, DROP query, DROP, DETACH, TRUNCATE, CHECK TABLE, KILL QUERY query, KILL, OPTIMIZE query, OPTIMIZE TABLE, WATCH query, WATCH, SHOW ACCESS query, SHOW ACCESS, ShowAccessEntitiesQuery, SHOW GRANTS query, SHOW GRANTS, SHOW PRIVILEGES query, SHOW PRIVILEGES, INSERT query, INSERT INTO, USE query, USE, SET ROLE or SET DEFAULT ROLE query, SET ROLE DEFAULT, SET ROLE, SET DEFAULT ROLE, SET query, SET, SYSTEM query, SYSTEM, CREATE USER or ALTER USER query, ALTER USER, CREATE USER, CREATE ROLE or ALTER ROLE query, ALTER ROLE, CREATE ROLE, CREATE QUOTA or ALTER QUOTA query, ALTER QUOTA, CREATE QUOTA, CREATE ROW POLICY or ALTER ROW POLICY query, ALTER POLICY, ALTER ROW POLICY, CREATE POLICY, CREATE ROW POLICY, CREATE SETTINGS PROFILE or ALTER SETTINGS PROFILE query, ALTER SETTINGS PROFILE, ALTER PROFILE, CREATE SETTINGS PROFILE, CREATE PROFILE, CREATE FUNCTION query, DROP FUNCTION query, CREATE NAMED COLLECTION, DROP NAMED COLLECTION query, Alter NAMED COLLECTION query, ALTER, CREATE INDEX query, DROP INDEX query, DROP access entity query, GRANT or REVOKE query, REVOKE, GRANT, EXTERNAL DDL query, EXTERNAL DDL FROM, TCL query, BEGIN TRANSACTION, COMMIT, ROLLBACK, SET TRANSACTION SNAPSHOT, Delete query, DELETE, BACKUP or RESTORE query, BACKUP, RESTORE. (SYNTAX_ERROR) (version 23.3.1.2399 (official build))`

### Package version

^2.0.1

### NestJS version

^9.0.0

### Node.js version

v16.14.0

### In which operating systems have you tested?

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by depyronick** [comment]
@levkovich2806 

`CREATE TABLE` is not an `INSERT`. You should use `query` or `queryPromise` instead of `insertPromise`.

**Comment by MrTomSawyer** [comment]
@levkovich2806 Have you resolve the issue? I have the same stuff here bruh

**Comment by depyronick** [comment]
should be fixed with 2.0.2

@MrTomSawyer 
@levkovich2806 

please check and close the issue if it fixes the issue

**Comment by depyronick** [comment]
closing due to inactivity, feel free to reopen if still occurs
